src/v/pandaproxy/schema_registry/types.h

```diff
@@ -315,6 +315,11 @@ struct seq_marker {
     schema_version version;
     seq_marker_key_type key_type{seq_marker_key_type::invalid};
 
+    bool operator==(const seq_marker& other) const {
+        return key_type == other.key_type && seq == other.seq
+               && node == other.node;
+    }
```

Comment:
I think the general case for `operator==` should include the version
```suggestion
    friend bool operator==(const seq_marker&, const seq_marker&) = default;
```

Reply:
😍